### DISCUSSION TITLE
Mobile Architecture Review Groups

### DISCUSSION THREAD

**Discussion by mathonsunday** [MobileNativeFoundation/discussions#124] [General]
Hey everyone 👋  

The mobile team at Compass grew considerably over the last few years to 30+ engineers. Our Mobile Infra team owns tooling, security, CI, releases, UI testing infrastructure, evaluation of new architectures and technologies and so on. 

I see an opportunity to establish a Mobile Architecture Review Group to  drive the evaluation and adoption of new architectures and technologies. This group would consist of individual contributors (ICs) who are either very senior already or show promise and interest in these topics. Over the long term, this group would handle deciding on and documenting platform wide best practices. 

My questions for the community:
1. Does your mobile team have such a group? If so, is it going well? 
2. If not, what are you doing instead?

**Comment by mkattima1** [comment]
@mathonsunday , At Intuit, we have RFC review process which helps drive mobile architecture, features and anything else that impacts mobile community in the company. Let's connect, I can provide you more details.

**Comment by sberrevoets** [comment]
Lyft has a group of Mobile Tech Leads which is similar to the review group you're talking about. While purely architectural decisions are mostly made by a dedicated architecture team (in collaboration with other teams in our Mobile Infra org), the Mobile Tech Leads to a large extent define the overall landscape of technologies and processes we use.

They work through working groups that any mobile engineer can be involved in, so that anybody in the mobile community at Lyft can be involved if they want to be. Examples of these groups are documentation, interviewing, onboarding, etc.

**Reply by mkattima1** [reply]
Thanks for sharing @sberrevoets , I am curious to learn more. It will be great to discuss on this further together and share best practices. Would it possible to meet with relevant folks from Lyft? I'll bring in my team to the discussion.
